susemanager-utils/susemanager-sls/salt/switch_to_bundle/mgr_switch_to_venv_minion.sls

```diff
@@ -0,0 +1,19 @@
+{%- set avoid_venv_salt_minion = salt['pillar.get']('custom_info:mgr_avoid_venv_salt_minion') %}
+{%- set is_salt_ssh = salt['pillar.get']('contact_method') in ['ssh-push', 'ssh-push-tunnel'] %}
+{%- set is_saltboot = salt['file.file_exists']('/etc/ImageVersion') %}
+
+{%- if grains['saltversion'] == '3000' and not is_salt_ssh and 
+       not avoid_venv_salt_minion == 'True' and not is_saltboot %}
```

Comment:
Shouldn't this be like:

```suggestion
       not avoid_venv_salt_minion == True and not is_saltboot %}
```

Reply:
Done (I just used the lowercase version of the `true` literal in Jinja).